src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background: #e5e5e5;
+}
+
+.card {
+  width: 200px;
+  padding: 15px 14px;
+  margin: 20vh auto;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.imac-image {
+  height: 134px;
+  width: 160px;
+  margin: 15px 0 0 5px;
+}
+
+.card-heading {
+  margin-top: 41px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card-link {
+  display: block;
+  cursor: pointer;
+  text-decoration: none;
+  color: #000;
+}
+
+.card-link:hover {
+  text-decoration: underline;
+}
+
+.card-code {
+  font-size: 11px;
+  margin-top: 5px;
+}
+
+.rate__block {
+  height: 15px;
+  margin: 17px 0 0 1px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__block {
+  display: flex;
+}
+
+.star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background: url(images/star.svg) no-repeat;
+}
+
+.star:nth-child(-n+4) {
```

Comment:
add classes for all cases(-n+1, -n+2 etc). this cards should be an example of working scenario, not hardcoded only for four stars. use your work from `stars` task